src/stdatamodels/jwst/transforms/resources/schemas/stsci.edu/jwst_pipeline/miri_wfss_dispersion-1.0.0.yaml

```
@@ -0,0 +1,48 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/jwst_pipeline/miri_wfss_dispersion-1.0.0"
+title: >
+  MIRI WFSS dispersion model
+description: |
+  Supports one model:
+   - Given x, y, wave, order in direct image return x,y,wave,order in dispersed
+   - Given x, y, x0, y0, order in dispersed image returns x, y, wave, order in the direct image.
+anyOf:
```

```suggestion
allOf:
```
Is there a reason for using `anyOf` instead of `allOf` here? Using `anyOf` would mean that if the instance passed validation again `transform-1.4.0` the reset of the schema contents would be ignored. `allOf` considers both subschemas (`transform-1.4.0` and the additional schema contents).